account_cash_invoice/models/account_bank_statement.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2021 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class AccountBankStatement(models.Model):
+
+    _inherit = "account.bank.statement"
+
+    def button_post(self):
+        lines_of_moves_to_reconcile = self.line_ids.filtered(
+            lambda line: line.move_id.state != "posted" and line.invoice_id
+        )
+        super(AccountBankStatement, self).button_post()
```

Comment:
Minor change:
You should save the result (even if it's `None`) and return it at the end.
```
result = super().button_post()
...
return result
```

Reply:
Thank you! Done :smile: